src/performance-tracking/types/PerformanceMetric.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Enum with all performance metrics events strings
+ */
+enum PerformanceMetric {
```

Comment:
IIRC we don't like enums

Reply:
I was about to ask "but why" 😄 
Thanks I will read that and analyse it. Hopefully I will learn something new